product_docs/docs/eprs/6.2/07_common_operations/11_using_ssl_connections.mdx

```diff
@@ -261,7 +261,7 @@ The encrypted password must be specified with the `sslTrustStorePassword` config
 Encrypt the password using the xDB Replication Server CLI encrypt command. The following example shows this process encrypting the password contained in file infile.
 
 ```text
-$ export PATH=/usr/lib/jvm/java-1.7.0-openjdk-1.7.0.45.x86_64/jre/bin:$PATH
+$ export PATH=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.45.x86_64/jre/bin:$PATH
```

Comment:
Is "java-1.8.0-openjdk-1.8.0.45.x86_64" correct? Other references (earlier and later) to the Java 8 path say "java-.0-openjdk-1.8.0.322.b06-1.el7_9.x86_64".

Reply:
This is correct and the other instances are wrong. I thought I had caught them all.